view/frontend/web/js/view/payment/method-renderer/bnpl-method.js

```diff
@@ -37,16 +37,6 @@ define(
                     document.body.appendChild(klumpCheckout);
                 }
 
-                let script = document.createElement('script');
```

Comment:
@richienabuk Wouldn't removing this affect the checkout?

Reply:
The script is added in the head section now using default template file. I added in the template to avoid duplicating it and making it hard to maintain.